kafka_topic_enforcer/src/main/java/com/tesla/data/topic/enforcer/TopicEnforcer.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (c) 2020. Tesla Motors, Inc. All rights reserved.
+ */
+
+package com.tesla.data.topic.enforcer;
+
+import static tesla.shade.com.google.common.base.Preconditions.checkArgument;
+
+import com.tesla.data.enforcer.Enforcer;
+import com.tesla.data.topic.enforcer.ConfigDrift.Result;
+import com.tesla.data.topic.enforcer.ConfigDrift.Type;
+
+import io.prometheus.client.Gauge;
+
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/** Enforce configured topics state on a kafka cluster. */
+public class TopicEnforcer extends Enforcer<ConfiguredTopic> {
```

Comment:
Not clear why the diff doesn't show this as `rename`, but this is just stripped down version of `Enforcer`. It has just the `alter` methods.